elasticsearch.md

```diff
@@ -912,55 +913,113 @@ readonlyrest:
 
 ### Kibana-related rules
 
-#### `kibana_access`
+#### `kibana`
 
-`kibana_access: ro`
+```yaml
+kibana:
+  access: ro # required
+  kibana_index: ".kibana_custom_index" # optional
```

Comment:
@sscarduzio I recall we leave the decision about naming for later. So, it's now I think :)

Do we leave it as it's or do we change to:
```
kibana:
  access: to
  index: ".kibana_custom_index"
  template_index: ".kibana_template"
```
?

Reply:
yes the rocket in my head was like "yes change it". It was not clear at all 🤣

Reply:
done